deploy/unifi.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env sh
+
+#Here is a script to deploy cert to unifi controller. Based on:
+# https://community.ubnt.com/t5/UniFi-Wireless/Your-own-SSL-key-and-cert/m-p/484943#M39260
+
+#returns 0 means success, otherwise error.
+
+########  Public functions #####################
+
+#domain keyfile certfile cafile fullchain
+unifi_deploy() {
+  _cdomain="$1"
+  _ckey="$2"
+  _ccert="$3"
+  _cca="$4"
+  _cfullchain="$5"
+
+  _debug _cdomain "$_cdomain"
+  _debug _ckey "$_ckey"
+  _debug _ccert "$_ccert"
+  _debug _cca "$_cca"
+  _debug _cfullchain "$_cfullchain"
+
+  if ! _exists keytool; then
+    _err "keytool not found"
+    return 1
+  fi
+
+  _pkcs12="$(_mktemp)"
+  _keypass="${DEPLOY_UNIFI_KEYPASS:-aircontrolenterprise}"
```

Comment:
please use `_savedomainconf()` function to save the "DEPLOY_UNIFI_KEYPASS" and  "DEPLOY_UNIFI_KEYSTORE".

See example:
https://github.com/Neilpang/acme.sh/blob/master/deploy/exim4.sh#L92